pkg/chunk/testutils/testutils.go

```diff
@@ -84,9 +90,13 @@ func dummyChunkFor(now model.Time, metric labels.Labels) chunk.Chunk {
 		now,
 	)
 	// Force checksum calculation.
-	err := chunk.Encode()
+	err = chunk.Encode()
 	if err != nil {
 		panic(err)
 	}
 	return chunk
 }
+
+func TeardownFixture(t *testing.T, fixture Fixture) {
```

Comment:
Is there any specific reason why this function was introduced? It looks a bit superfluous to me, given we can directly call `require.NoError(t, fixture.Teardown())` instead of `TeardownFixture()`.

Reply:
It was done in this commit. 8bf4365 It was just something i noticed we were doing 2 places so felt like a natural thing since the fixture was already in the testutils.go class. I could go either way....ill leave that up to you but that was the reasoning.